src/main/java/org/broadinstitute/consent/http/db/mapper/DarCollectionSummaryReducer.java

```diff
@@ -45,17 +49,26 @@ public void accumulate(Map<Integer, DarCollectionSummary> map, RowView rowView)
         //ignore exception, it means dar_status and dar_reference_id wasn't included for this query
       }
 
-      election = rowView.getRow(Election.class);
-      if (Objects.nonNull(election.getElectionId())) {
-        summary.addElection(election);
-        summary.addDatasetId(election.getDataSetId());
+      try{
```

Comment:
formatting nit: Here and on line 62, add a space between the try and the brace
```suggestion
      try {
```
Intellij should auto format this for you with command-shift-L